### DISCUSSION TITLE
Using TipTap with Vuejs 3 and Axios

### DISCUSSION THREAD

**Discussion by peterdpolet** [ueberdosis/tiptap#2378] [General Questions]
I'm trying to store/retrieve text edited in Tiptap  in postgres using Vuejs 3, Vuex and axios. I'm struggling to get the downloaded text into a Tiptap editor. I think its because of the async call from axios returning a promise which is then not updated by the actual content. Has anyone got this working or knows of an example?
Thanks!

**Comment by hanspagel** [comment]
You can try using setContent:

https://tiptap.dev/api/commands/set-content
